src/components/Button/Button.scss

```diff
@@ -277,16 +308,29 @@ $partial-button-filled-pressed-box-shadow: inset 0 0 0 0 transparent,
     @include high-contrast-button-outline;
   }
 
+  &:focus:not(:active) > .Content {
+    @include focus-ring;
+  }
+
   &.pressed > .Content {
     @include plain-button-backdrop(rgba(color('ink', 'lighter'), 0.1));
   }
 
   &.pressed:hover:not(.iconOnly) > .Content,
-  &.pressed:active:not(.iconOnly) > .Content,
-  &:focus:not(.iconOnly) > .Content {
+  &.pressed:active:not(.iconOnly) > .Content {
     @include plain-button-backdrop;
   }
 
+  &:focus:not(.iconOnly) > .Content {
+    @include plain-button-backdrop(
```

Comment:
This and the below `:active` selector add padding and negative margin to styles to those states which previous had none. I noticed in Firefox this causes the text to wrap if it is at the edge of its container:

![firefox wrap](https://user-images.githubusercontent.com/344839/67260117-6e7fb380-f44e-11e9-935f-e3a205a85aec.gif)